.vscode/settings.json

```diff
@@ -38,5 +38,6 @@
             ],
             "url": "/ext/data/schemas/recommended-dictionaries-schema.json"
         }
-    ]
```

Comment:
this is so vscode uses the workspace version of ts and doesn't yell at you about stuff that's erroring in the latest version